version/description

```diff
@@ -1,9 +1,7 @@
-v0.5.0
-
+v0.5.1
 Features:
-* Link pods to priority classes #42
-* Rename "master" API #41 to use inclusive language
-
+* install-cni, Add resources requests for initContainer
+* dependencies: use containernetworking/plugins v0.9.1
 ```
-docker pull quay.io/kubevirt/macvtap-cni:v0.5.0
+docker pull HANDLER_IMAGE
```

Comment:
@maiqueb @phoracek 
what is the process please?
can you assist in creation of quay.io/kubevirt/macvtap-cni:v0.5.1 ?

Reply:
You'll have to manually make this right, I guess that something is not correctly replacing the place holder with the actual name of the image. 

Update it manually so it features the img you're creating - i.e. `quay.io/kubevirt/macvtap-cni:v0.5.1` .

Reply:
ah ok and then just the CI will create it ?

Reply:
yes.

Reply:
thanks, done